### ISSUE TITLE
VST crash with kernel 5.2

### ISSUE THREAD

**Issue by olivierlemoal** [osxmidi/LinVst#86]
FYI, some VST (S-Gear / TH-U) are crashing (page fault) when loaded with LinVst since Linux 5.2 upgrade (I'm using Arch Linux). Downgrading to 5.1.15 solves this issue. I'm not really sure it's LinVst related through.

**Comment by osxmidi** [comment]
Thanks, I'll try to look into it.

**Comment by Ogami-Z** [comment]
I can confirm this is also happening on Manjaro with kernel 5.2 (5.2.2) all previous installed kernels are fine. Tested with Kontakt, Amplitube & Replika

**Comment by osxmidi** [comment]
I installed kernel 5.2.3 and there was a crash involving a new Wine memory allocation.

So it looks to me like there is some 5.2 kernel compatibility issue and Wines memory allocation.

Anyway I think I've got a workaround that seems to work ok on my system and I've made the changes.

**Comment by olivierlemoal** [comment]
Fix works for me on 5.2.2 :+1:

**Comment by Ogami-Z** [comment]
I just had a chance to check out the updated version & it does initially seem to fix the problem. That is until you open & close a plugin in Reaper 3 or 4 times then Reaper freezes & has to be killed along with any still running wine or linvst in the background.

Tested with Kontakt 5, Amplitube 4 & various fabfilter plugins. Just on the off chance would it have anything to do with the lin-vst-servertrack.exe lin-vst-servertrack.exe.so. I noticed there aren't any others included in 2.55like previous versions.

For now I've had to revert back from kernel 5.2.2 to 5.1.19 & reconvert all plugins with linvst 2.32.
